packages/translate/lib/notStartedReason.tsx

```diff
@@ -126,6 +126,10 @@ export function translateNotStartedReason(code: NotStartedReason): Translation {
 			return {
 				title: <text>SuitestDrive can't launch NetworkLog service on Mac</text>,
 			};
+		case NOT_STARTED_REASON.DEVICE_OFFLINE:
+			return {
+				title: <fragment>Device could not be reached, please try again later. Contact <link href="mailto:support@suite.st">support</link> if problem persists.</fragment>,
```

Comment:
Please, update:
* "please try again later" => "please try again **in a few minutes**"
* "if problem persists" => "if **the** problem persists"